pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -10,7 +10,7 @@
 
 from homeassistant.const import Platform
 
-DEVICE_CLASS = object()
+DEVICE_CLASS = "***DeviceClass"
```

Comment:
This change makes it prettier:

`W7432: Return type should be [<object object at 0x7f96a5358b00>, 'str', None] (hass-return-type)`
versus
`W7432: Return type should be ['***DeviceClass', 'str', None] (hass-return-type)`